### PR TITLE
Mk migrate docs to hangzhou

### PR DESCRIPTION
**Pull request by michaelkernaghan**
Updated the Docs live code examples to work with Hangzhounet by originating the required contracts on hangzhounet

updated references to Granada to Hangzhou

fixed some missed references to Florence, now are Hangzhou

originated the contracts needed by Examples for H.

See https://ecadlabs.ontestpad.com/script/169#//
and  https://docs.google.com/document/d/15rfuo70GkMHiKV_ChrRUbLcxOUo7S6hegQ4pzX21WlQ/edit#heading=h.4i1h3h5p804x

### CODE REVIEW

File: integration-tests/contract-fetch-multiple-big-map-keys.spec.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
                 }
             });
             const contract = await op.contract();
+            console.log("contractAddress : "+ contract.address);
```
Comment (roxaneletourneau) [inline integration-tests/contract-fetch-multiple-big-map-keys.spec.ts:35]: I think you forgot to remove the console.log here

File: integration-tests/contract-originate-token-contract.spec.ts [hunk 2]
```diff
@@ -28,6 +28,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
       expect(op.hash).toBeDefined();
       expect(op.includedInBlock).toBeLessThan(Number.POSITIVE_INFINITY)
       contract = await op.contract();
+      console.log("contract address : "+contract.address)
```
Comment (roxaneletourneau) [inline integration-tests/contract-originate-token-contract.spec.ts:31]: I think you forgot to remove the console.log here

File: integration-tests/contract-permits.spec.ts [hunk 3]
```diff
@@ -36,6 +36,8 @@ CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
       expect(op.hash).toBeDefined();
       expect(op.includedInBlock).toBeLessThan(Number.POSITIVE_INFINITY);
       const permit_contract = await op.contract();
+      
+      console.log("permit contract address:"+permit_contract.address)
```
Comment (roxaneletourneau) [inline integration-tests/contract-permits.spec.ts:40]: I think you forgot to remove the console.log here

File: integration-tests/lambda-view.spec.ts [hunk 4]
```diff
@@ -47,6 +47,8 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 
       await op.confirmation()
       const contract = await op.contract();
+      
+      console.log("lambda_view #1 contract address: "+ contract.address)
```
Comment (roxaneletourneau) [inline integration-tests/lambda-view.spec.ts:51]: I think you forgot to remove the console.log here

File: integration-tests/lambda-view.spec.ts [hunk 5]
```diff
@@ -142,6 +144,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 
       await op.confirmation()
       const contract = await op.contract();
+      console.log("lambda_view #2 contract address: "+ contract.address)
```
Comment (roxaneletourneau) [inline integration-tests/lambda-view.spec.ts:147]: I think you forgot to remove the console.log here

**Review by roxaneletourneau — CHANGES_REQUESTED** [review]

File: docs/complex_parameters.md [hunk 6]
```diff
@@ -171,7 +171,7 @@ importKey(Tezos, emailExample, passwordExample, mnemonicExample, secretExample)
     println(`Waiting for ${op.hash} to be confirmed...`);
     return op.confirmation(1).then(() => op.hash);
 }).then(hash => {
-    println(`Operation injected: https://better-call.dev/granadanet/KT1Ho26P3cnTjeLp29VPuHj2rLGfZeUKTdMo/operations`);
+    println(`Operation injected: https://better-call.dev/hangzhounet/KT1CfFBaLoUrgv93k8668KCCcu2hNDNYPz4L/operations`);
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply docs/complex_parameters.md:174]: fixed.

File: docs/making_transfers.md [hunk 7]
```diff
@@ -46,7 +46,7 @@ fetch('https://api.tez.ie/keys/granadanet/', {
     render(`Waiting for ${op.hash} to be confirmed...`);
     return op.confirmation(1).then(() => op.hash);
   })
-  .then((hash) => render(`Operation injected: https://florencenet.tzstats.com/${hash}`))
+  .then((hash) => render(`Operation injected: https://hangzhounet.tzstats.com/${hash}`))
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply docs/making_transfers.md:49]: fixed

File: docs/metadata-tzip16.md [hunk 8]
```diff
@@ -215,7 +215,7 @@ Try to run the view:
 
 Tezos.addExtension(new Tzip16Module());
 
-const contractAddress = "KT1LhUqpUGfxZaJFMrxAZ6WNAuM8FPjDLBcE";
+const contractAddress = "KT1Vms3NQK8rCQJ6JkimLFtAC9NhpAq9vLqE";
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply docs/metadata-tzip16.md:218]: fixed.

File: docs/metadata-tzip16.md [hunk 9]
```diff
@@ -243,7 +243,7 @@ Try to run the view:
 
 Tezos.addExtension(new Tzip16Module());
 
-const contractAddress = "KT1GqWXHeeJHEHVPcqJYfKDuHkdpkJevjeAc";
+const contractAddress = "KT19rDkTYg1355Wp1XM5Q23CxuLgRnA3SiGq";
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply docs/metadata-tzip16.md:246]: fixed

File: docs/quick_start.md [hunk 10]
```diff
@@ -138,17 +138,17 @@ Tezos.contract
...
   })
-  .then((hash) => println(`Operation injected: https://granada.tzstats.com/${hash}`))
+  .then((hash) => println(`Operation injected: https://hangzhou.tzstats.com/${hash}`))
   .catch((error) => println(`Error: ${error} ${JSON.stringify(error, null, 2)}`));
 ```
 
 ### Interact with a smart contract
 
-Calling smart contract operations requires a configured signer; in this example we will use a faucet key. The Ligo source code for the smart contract [KT1WsSQ4BCb9PoBRtxvH9fys2j257uaxquro][smart_contract_on_better_call_dev] used in this example can be found in a [Ligo Web IDE][smart_contract_source].
+Calling smart contract operations requires a configured signer; in this example we will use a faucet key. The Ligo source code for the smart contract [KT1Gx18RNNvFvrv1gb9abBWCqxEPyFT5duTP][smart_contract_on_better_call_dev] used in this example can be found in a [Ligo Web IDE][smart_contract_source].
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply docs/quick_start.md:147]: replaced with the correct contract address. fixed.

File: integration-tests/contract-fetch-multiple-big-map-keys.spec.ts (hunk 1 above)
Comment (michaelkernaghan) [reply integration-tests/contract-fetch-multiple-big-map-keys.spec.ts:35]: fixed

File: integration-tests/contract-originate-token-contract.spec.ts (hunk 2 above)
Comment (michaelkernaghan) [reply integration-tests/contract-originate-token-contract.spec.ts:31]: fixed

File: integration-tests/contract-permits.spec.ts (hunk 3 above)
Comment (michaelkernaghan) [reply integration-tests/contract-permits.spec.ts:40]: fixed

File: integration-tests/lambda-view.spec.ts (hunk 4 above)
Comment (michaelkernaghan) [reply integration-tests/lambda-view.spec.ts:51]: fixed

File: integration-tests/lambda-view.spec.ts (hunk 5 above)
Comment (michaelkernaghan) [reply integration-tests/lambda-view.spec.ts:147]: fixed

**Comment by michaelkernaghan** [comment]
I have replaced this PR with another one.

**Comment by michaelkernaghan** [comment]
I should not have closed this one
I will resolve the merge conflicts

**Comment by roxaneletourneau** [comment]
replaced by https://github.com/ecadlabs/taquito/pull/1188

**Pull request closed without merging**